tests/sampleData/metadata.ts

```diff
@@ -101,4 +101,34 @@ export const sampleMetadata = [
     deleted: true,
     updated_at: new Date('2023-04-06T15:25:40.000Z'),
   },
+  {
+    _id: new ObjectId('646bae50c41d7b9a472b6082'),
+    project: 'landing',
+    branch: 'test-same-branch',
+    github_username: 'beepboop-metadata',
+    title: 'MongoDB Documentation',
+    eol: false,
+    slugToTitle: {},
+    toctreeorder: {},
+    parentPaths: {},
+    static_files: {},
+    build_id: new ObjectId('642ec854c38bedd45ed31238'),
+    created_at: new Date('2023-04-07T19:00:00.000Z'),
+    updated_at: new Date('2023-04-08T14:00:00.000Z'),
+  },
```

Comment:
Maybe overkill, but could we add a metadata document that has the same `project` and `branch` name as one of these, but a different user?

Reply:
Done! Tests seem to pass without any snapshot updates. The new test should have caught the second `landing` project if it was an issue 👍